CONTRIBUTING.md

```diff
@@ -115,6 +115,13 @@ $ git checkout -b my-new-feature
 
 Make that feature or bugfix awesome!
 
+## Configuration changes
+
+If you add any configuration or change any configuration, 
+you have to update `docs/basics/janusgraph-cfg.md`. 
+This should be done by running `mvn --quiet clean install -DskipTests=true -pl janusgraph-doc -am`, 
+otherwise documentation will fail on Travis-CI.
```

Comment:
Isn't the real problem that we don't have up to date docs about our config in the repo if this isn't done?